### PR TITLE
Witch Water Improvements and Tainted Sandstone

### PR DESCRIPTION
**Pull request by Shnupbups**
- Tainted Sand can now be made into Tainted Sandstone, which has all the variants that regular or red Sandstone has
- Witch Water Buckets now have a texture and a translation
- Witch Water now gives its potion effects to every living thing except for the undead and Pumpcowns
- Witch Water now only turns Villagers into Witches if the difficulty isn't set to Peaceful
- Witch Water now turns Cows into Pumpcowns
- Witch Water now turns Overworld blocks into their Spooky Dimension equivalents when they are dropped in

### CODE REVIEW

**Review by ImAK9 — APPROVED** [review]
Seems fine to me! Nice!

**Review by Juuxel — CHANGES_REQUESTED** [review]
The recipes are indented with 2 spaces, and it should be either tabs or 4 spaces.

**Comment by Shnupbups** [comment]
Whoops, I copied them from the vanilla ones and forgot to change the indentation

**Comment by Juuxel** [comment]
Doesn't vanilla have 4 spaces?

**Comment by Shnupbups** [comment]
Nope, it uses 2 spaces for jsons.

File: src/main/resources/assets/spookytime/lang/de_de.json [hunk 1]
```diff
@@ -1,32 +1,28 @@
...
+	"spookytime.about.github": "Unserer Githublink liegt da: ",
+	"spookytime.github": "https://github.com/fabric-community/spooky-time",
+	"spookytime.cmd.spooktober": "Es ist Spooktober!",
+	"spookytime.cmd.tillspooktober": "Es ist %d Tage bis Spooktober",
+	"item.spookytime.pumpcown_spawn_egg": "Pumpcown-Spawn-Ei",
+	"item.spookytime.blaze_skirt": "Lohenrock",
+	"item.spookytime.spookium_ingot": "Spookiumbarren",
+	"item.spookytime.spookium_nugget": "Spookiumklumpen",
+	"item.spookytime.soul_bottle": "Seeleflasche",
+	"item.spookytime.pumpkin_candy": "K�rbisbonbon",
```
Comment (Juuxel) [inline src/main/resources/assets/spookytime/lang/de_de.json]: Did the file encoding change? These question marks were `ü` before.

Comment (Shnupbups) [reply src/main/resources/assets/spookytime/lang/de_de.json]: Hmm, odd. All I did was run them through Idea's Reformat Code. I'll see if I can fix.

**Review by Juuxel — APPROVED** [review]

**Pull request merged**